src/App.tsx

```diff
@@ -1,156 +1,100 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { FilterPanel } from './FilterPanel';
+import { InputForm } from './InputForm';
+import { Notifications } from './Notifications';
+import { TodoList } from './TodoList';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 10592;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
-
-  return (
-    <div className="todoapp">
-      <h1 className="todoapp__title">todos</h1>
-
-      <div className="todoapp__content">
-        <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
-
-          {/* Add a todo on form submit */}
-          <form>
-            <input
-              type="text"
-              className="todoapp__new-todo"
-              placeholder="What needs to be done?"
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [removeTodoIsClicked, setRemoveTodoIsClicked] = useState(false);
+  const [addTodoIsClicked, setAddTodoIsClicked] = useState(false);
+  const [editTodoIsClicked, setEditTodoIsClicked] = useState(false);
+  const [filterMode, setFilterMode] = useState<string>('All');
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+
+  const showNotification = removeTodoIsClicked
+    || addTodoIsClicked || editTodoIsClicked;
+
+  const getTodos = async () => {
+    const data = await client.get('/todos?userId=10592') as Todo[];
+
+    setTodos(data);
+  };
+
+  useEffect(() => {
+    getTodos();
+  }, []);
+
+  const filteredTodosArr = useMemo(() => {
+    return todos.filter((todo) => {
+      if (filterMode === 'Active') {
+        return !todo.completed;
+      }
+
+      if (filterMode === 'Completed') {
+        return todo.completed;
+      }
+
+      return todos;
```

Comment:
Return 'todo' instead of 'todos' in the filter function to ensure each todo is returned